ci-operator/config/openshift/cluster-api-provider-azure/openshift-cluster-api-provider-azure-master.yaml

```diff
@@ -71,6 +79,11 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure
+- as: e2e-hypershift
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: hypershift
+    workflow: hypershift-aws-e2e-external
```

Comment:
The workflow does say aws here, is this expected?

Reply:
no, we'll need to setup an external for aks.
/hold